src/app/pages/main/components/community-preview/community-preview.component.html

```diff
@@ -1,41 +1,12 @@
 <div fxLayout="column">
   <mat-form-field>
-    <input
-      placeholder="Search communities"
-      type="text"
-      [formControl]="filter"
-      matInput
-      autocomplete="off"
-    />
+    <input placeholder="Search communities" type="text" [formControl]="filter" matInput autocomplete="off" />
   </mat-form-field>
   <div class="community-preview-list-container">
     <cdk-virtual-scroll-viewport itemSize="110" class="community-preview-list-view-port">
-      <div
-        class="community"
-        fxLayout="row"
-        fxLayoutAlign="space-evenly"
-        *cdkVirtualFor="let community of filtered$ | async"
-        (click)="onSelected(community.name)"
-      >
-        <picture>
-          <source
-            *ngFor="let media of responsiveImagesConfiguration"
-            [media]="'(max-width: ' + media.max + 'px) and (min-width: ' + media.min + 'px)'"
-            [srcset]="'./assets/images/' + community.id + media.suffix + '.' + media.format"
-            [type]="'image/' + media.format"
-          />
+      <ngcommunity-community-item *cdkVirtualFor="let community of filtered$ | async" [community]="community"
+        (communitySelected)="onSelected(community.name)"></ngcommunity-community-item>
```

Comment:
I'd use native `click` event

```suggestion
        (click)="onSelected(community.name)"></ngcommunity-community-item>
```

Reply:
Good catch!